### ISSUE TITLE
i can not start the project

### ISSUE THREAD

**Issue by wiwi111** [OMKE/ULA#62]
**Describe the bug**
docker-compose up -d 
not work

**To Reproduce**
1. docker-compose up -d 
2.  found 
``` 
Building main_app_backend
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /root/ULA/backend/ULA-1/Dockerfile: no such file or directory
ERROR: Service 'main_app_backend' failed to build
```

Thank you

**Comment by OMKE** [comment]
Docker containers are not supported, I made a docker compose file while the app was still a monolith.
If you have time to do it, feel free to make a pull request.

**Comment by hcalebe** [comment]
I think it is the ULA-1 directory which is missing from the backend directory

**Comment by OMKE** [comment]
> I think it is the ULA-1 directory which is missing from the backend directory

Yes, the docker compose file is not updated for this architecture. I don't have time to do it, but if you're willing to dockerize it, please feel free to make a PR.
